.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -141,7 +141,7 @@ jobs:
 
           CIBW_BEFORE_TEST: |
             python -m pip install -r requirements-tests.txt
-            if ${{ matrix.pl_backend == 'lightning_kokkos'}}; then SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" pip install -e . -vv; fi
+            if ${{ matrix.pl_backend == 'lightning_kokkos'}}; then SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" python -m pip install . -vv; fi
```

Comment:
Just curious, did you try?
```python -m pip install -e . -vv```
Because I can see how the absence of `python -m` could make things go wrong.

Reply:
Yes, @maliasadi noticed `python -m` was missing in a few places, but adding it didn't impact the failures. I'm still adding it here for good measure.

Reply:
Cool.